torch/_higher_order_ops/wrap.py

```diff
@@ -87,6 +87,41 @@ def wrapper():
 wrap_with_autocast = WrapWithAutocast()
 
 
+class DynamoBypassingWrapper(HigherOrderOperator):
+    def __init__(self):
+        super().__init__("dynamo_bypassing_wrapper")
+
+    def __call__(
+        self,
+        *args,
+        **kwargs,
+    ):
+        # Dynamo already traces the body of HigherOrderOp beforehand when it
+        # so no need to trace into it.
+        import torch._dynamo  # noqa: F401
+        from torch._dynamo import disable
+
+        is_compiling = (
```

Comment:
Is this accurate? An alternative is:
```
How about we do something like this:
```python
def __call__(sefl, wrapper_fn_or_key, inner_fn, *inner_args, **inner_kwargs):
  is_compiling = isinstance(wrapper_fn_or_key,  str)
  if is_compiling:
    assert isinstance(inner_fn, torch.fx.GraphModule)
    wrapper_fn = inner_fn.meta[wrapper_fn_or_key]
  eles:
    wrapper_fn = wrapper_fn_or_key
```

Reply:
Thanks this looks cleaner